sdcm/utils/cloud_monitor/report.py

```diff
@@ -30,7 +30,8 @@ def to_html(self):
         return self.render_template()
 
     def to_file(self):
-        with tempfile.NamedTemporaryFile(prefix='cloud-report_', delete=False, suffix='.html') as report_file:
+        with tempfile.NamedTemporaryFile(mode="w", encoding="utf-8",
```

Comment:
the name of this method is not completely clear.

Reply:
It's a built-in, a bit hard to change it :)